### ISSUE TITLE
read.ctd.woce() should separate bottle oxygen from CTD oxygen

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#2090]
In working on [github.com/dankelley/ocencdf,](https://github.com/dankelley/ocencdf/issues/6), I realized that `oce::read.ctd.woce()` does not distinguish between CTD oxygen and bottle oxygen.  Considering bottle data to be subsidiary to CTD data, I think oce should do for oxygen what it does for other things that can be measured either way: use `oxygenBottle` to indicate bottle data, leaving `oxygen` to indicate the CTD variety.  See the snapshot for the relevant lines of code (172 and 185).

<img width="1184" alt="Screenshot 2023-05-29 at 3 19 56 PM" src="https://github.com/dankelley/oce/assets/99469/3686f6ec-ac3d-4167-aef9-037b11df4e08">

**Comment by dankelley** [comment]
For reference, below is a snapshot from https://cchdo.github.io/hdo-assets/documentation/manuals/pdf/90_1/chap4.pdf.  (I refer to this URL without link in the oce docs, presumably because I found that the link failed on occasion.  And, when I check it now, I note that it refers to an appendix G although the file ends, somewhat abruptly, without any appendices.  Maybe the file got removed and then it was rescanned but examining only the first N pages.  This is not worth checking into.)

<img width="740" alt="Screenshot 2023-05-29 at 3 37 22 PM" src="https://github.com/dankelley/oce/assets/99469/4fb66dbc-e358-4f51-b678-e8da12cdb0b7">
